### ISSUE TITLE
Cannot find ice executable in newest HiCPro install directory

### ISSUE THREAD

**Issue by jaavedm** [ay-lab/FitHiChIP#35]
FitHiChIP_HiCPro.sh runs fine until it encounters its first use of ICE.

I'm getting the error:
`/d/ja/packages/FitHiChIP/FitHiChIP_HiCPro.sh: line 1353: /d/ja/packages/HiC-Pro_2.11.4/scripts/ice: No such file or directory`

The command "ice" is not found in my HiC-Pro_2.11.4/scripts directory. In my Hicpro installation directory, I can see the script folder, but under script folder there is `ice_norm.sh`. `ice_norm.sh` seems to be a wrapper for ice. The Hicpro github page recommends installing ice from python repo. I did this using pip, and ice is on my path.

Maybe a recommend fix would be to change FitHiChIP_HiCPro.sh line 1353 
From
`ICEExec=$HiCProBasedir'/scripts/ice'`

to

`ICEExec='ice'`

**Comment by ay-lab** [comment]
Hi @jaavedm 
Thanks for your query. Did you install HiCPro? The downloaded package does not contain "ICE" executable but once you install it, you'll see it.
To install HiCPro, please check its README. First, edit the config-install.txt file if necessary, then run "make configure" and "make install".

**Comment by jaavedm** [comment]
Hello. Thanks for your response. I did download and install the latest version of HiCPro. The version I downloaded is 2.11.4. The package was installed successfully; I was able to run it on my HiChip data. 
This version of HiCPro did not have ice included in it. However FitHichip has hardcoded to expect the `ice `executable to be installed under the script folder of the HiCPro installation directory. `ice `is not included in the HiCPro installation directory, at least in version 2.11.4.
